packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -286,6 +283,14 @@ const VideoPlayerBase = class extends React.Component {
 		 */
 		loading: PropTypes.bool,
 
+		/**
+		 * An object containing properties to be passed to media controls component.
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		mediaControlsProps: PropTypes.Object,
```

Comment:
I'd rather not expose this. It doesn't make a lot of sense as a public API when `MediaControls` itself isn't public so there's no way to know what to pass without diving into the private source. If we feel it's truly necessary to support testware deep within VideoPlayer as the ticket requests, I'd rather define a static attribute (e.g. `data-media-controls`) on the node they could use to find it.